docs/ops/generation/Range_4.md

```diff
@@ -0,0 +1,84 @@
+## Range<a name="Range"></a>
+
+**Versioned name**: *Range-4*
+
+**Category**: Generation
+
+**Short description**: *Range* operation generates a sequence of numbers according input values [start, stop) with a step.
+
+**Attributes**:
+
+* *output_type*
+
+  * **Description**: the output tensor type
+  * **Range of values**: u1, u8, u16, u32, u64, i8, i16, i32, i64, f16, f32, boolean, bf16
+  * **Type**: string
+  * **Default value**: None
+  * **Required**: *Yes*
+
+**Inputs**:
+
+* **1**: "start" - A scalar of type T1. **Required.**
+* **2**: "stop" - A scalar of type T2. **Required.**
+* **3**: "step" - A scalar of type T3. **Required.**
+
+**Outputs**:
+
+* **1**: A tensor with type specified by attribute *output_type*.
+
+**Types**
+
+* *T1*, *T2*, *T3*: any numeric type.
+
+**Detailed description**:
+
+*Range* operation generates a sequence of numbers starting from the value in the first input (`start`) up to but not including the value in the second input (`stop`) with a `step` equal to the value in the third input, according to the following formula:
+
+\f[
+For a positive `step`, start<=val[i]<stop, for a negative `step`, start>=val[i]>stop, where val[i]=start+i*step.
+\f]
+
```

Comment:
Please figure out Pytorch's behaviour for this operation. And specify Range for it if needed.
For TF it seems the current Range-1 is sufficient